.github/workflows/canary-integration-test.yml

```diff
@@ -29,10 +29,18 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: consider debugging
+      - name: consider debugging (tmate)
         uses: ./.github/workflows/tmate_debug
         with:
-          use-tmate: ${{ secrets.USE_TMATE }}
+          use-tmate: true
+          use-upterm: false
+          debug-ci: ${{ contains(github.event.pull_request.labels.*.name, 'debug-ci') }}
+ 
+      - name: consider debugging (upterm)
+        uses: ./.github/workflows/tmate_debug
+        with:
+          use-tmate: false
+          use-upterm: true
           debug-ci: ${{ contains(github.event.pull_request.labels.*.name, 'debug-ci') }}
```

Comment:
With a composite action, I don't know why we would still need to have 2 separate invocations of it. Can we mak this so we only invoke `conisder debugging` once, and the composite job automatically runs both tmate and upterm while we are all getting used to upterm?